src/GoogleAdWordsEventForwarder.js

```diff
@@ -393,6 +553,22 @@
                     return 'Can\'t initialize forwarder: ' + name + ', conversionId is not defined';
                 }
 
+                if (window.gtag && forwarderSettings.enableGtag == 'True') {
+                    const initialConsentState = getUserConsentState();
+
+                    if (initialConsentState) {
+                        var defaultConsentPayload =
+                            generateConsentStatePayloadFromMappings(
+                                initialConsentState,
+                                consentMappings,
+                                consentPayloadDefaults,
+                            );
+                        consentPayloadAsString = stringifyPayload(defaultConsentPayload);
```

Comment:
Instead of `consentPayloadAsString` do you think something like `latestConsentPayloadAsString` or `updatedConsentPayloadAsString` might be more specific? I don't really like those either tbh.  Something that denotes that it's something that would continue to update.  Or maybe just putting a comment where it is defined to explain it will be updated whenever the mapped consents change.

```suggestion
                        consentPayloadAsString = stringifyPayload(defaultConsentPayload);
```